products/facilities/facility-locator/data/hours-of-operation.md

```diff
@@ -1,24 +1,25 @@
-## Hours of Operation 
+## Hours of Operation
 
 This page will capture the variations in the ways hours of operations is being conveyed by the various VA facilities and the business decisions made to standardize the input.
 
 **(WIP: Pending input from business and content experts)**
 
 | Data input | Display |
 |---|---|
-| Please Call for Hours | Please call for hours. | 
+| Please Call for Hours | Please call for hours. |
 | 7:00am - Sunset | 7:00 a.m. - Sunset (timezone when available) |
 | 800AM-1200 | 8:00 a.m. - 12:00 a.m. (timezone when available) |
+| 800AM-1200, 100PM-400PM  | 8:00 a.m. - 12:00 p.m., 1:00 p.m - 4:00 p.m. (timezone when available) |
```

Comment:
```suggestion
| 8:00 a.m. - 12:00 p.m., 1:00 p.m - 4:00 p.m.   | 8:00 a.m. - 12:00 p.m., 1:00 p.m - 4:00 p.m. (timezone when available) | 
```

Reply:
I made this suggestion because the data coming in from the Facility API is the same as we are passing though 8:00 a.m. - 12:00 p.m., 1:00 p.m - 4:00 p.m.